docs/source/install.rst

```diff
@@ -30,10 +30,6 @@ dependencies:
   * pkg-config [only for Linux and Mac]
   * Ipopt [>= 3.13 on Windows]
   * Python 3.6+
-  * setuptools
-  * cython
-  * numpy
-  * scipy [optional]
```

Comment:
You've also removed scipy here as an optional dep. How does pyproject.toml handle that?

Reply:
My understanding is that the `scipy` interface of `cyipopt` is not related to the build process, so it has no place in build requiremetns.

Reply:
This list is a list of all build and run time dependencies (optional or not). I'd like to leave the list intact in the documentation.

Reply:
I have restored the diff here, but I find that docs confusing. They say:
> To begin installing from source you will need to install the following dependencies

So I don't understand them to be "build or run time depedencies" necessarily.

Furthermore, no versions are specified here, which is inconsistent with [what `setup.py` requires](https://github.com/mechmotum/cyipopt/blob/master/setup.py#L24).